src/app/portfolio/_components/ProjectCard.tsx

```
@@ -167,8 +221,12 @@ export default function ProjectCard({
           <h2 className="grow text-3xl font-semibold">{title}</h2>
           {isOwner && (
             <div className="flex items-center gap-1">
-              <button className="p-1 font-semibold text-brand-tertiary">수정</button>
-              <button className="p-1 font-semibold text-brand-tertiary">삭제</button>
+              <button onClick={handleEdit} className="p-1 font-semibold text-brand-tertiary">
+                수정
+              </button>
+              <button onClick={handleDeleteModal} className="p-1 font-semibold text-brand-tertiary">
+                삭제
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)